### PR TITLE
update weights

### PR DESCRIPTION
**Pull request by coderobe**

### CODE REVIEW

**Comment by coderobe** [comment]
i tried to use substrate-weight-compare to render a comparison for this but it seems like i can not compile it

**Comment by coderobe** [comment]
this will be backported to the release branch once reviewed & merged

File: runtime/polkadot/src/weights/pallet_scheduler.rs [hunk 1]
```diff
@@ -86,9 +86,9 @@ impl<T: frame_system::Config> pallet_scheduler::WeightInfo for WeightInfo<T> {
 	// Storage: Preimage PreimageFor (r:1 w:1)
 	// Storage: Preimage StatusFor (r:1 w:1)
 	fn on_initialize_resolved(s: u32, ) -> Weight {
-		(1_701_000 as Weight)
-			// Standard Error: 26_000
-			.saturating_add((19_301_000 as Weight).saturating_mul(s as Weight))
+		(3_124_000 as Weight)
+			// Standard Error: 25_000
+			.saturating_add((18_227_000 as Weight).saturating_mul(s as Weight))
```
Comment (chevdor) [inline runtime/polkadot/src/weights/pallet_scheduler.rs:91]: x2

File: runtime/polkadot/src/weights/pallet_utility.rs [hunk 2]
```diff
@@ -45,29 +45,27 @@ use sp_std::marker::PhantomData;
 pub struct WeightInfo<T>(PhantomData<T>);
 impl<T: frame_system::Config> pallet_utility::WeightInfo for WeightInfo<T> {
 	fn batch(c: u32, ) -> Weight {
-		(10_504_000 as Weight)
+		(4_943_000 as Weight)
 			// Standard Error: 2_000
-			.saturating_add((4_057_000 as Weight).saturating_mul(c as Weight))
+			.saturating_add((3_798_000 as Weight).saturating_mul(c as Weight))
 	}
```
Comment (chevdor) [inline runtime/polkadot/src/weights/pallet_utility.rs:51]: / 2

**Comment by coderobe** [comment]
I *think* some block weights got skipped, or at least it appears so looking at [the logs](https://gitlab.parity.io/parity/mirrors/polkadot/-/jobs/1557066) 

rel:
```
2022-05-11 19:11:35 👶 Creating empty BABE epoch changes on what appears to be first startup.    
2022-05-11 19:11:35 Essential task `txpool-background` failed. Shutting down service.    
2022-05-11 19:11:35 CurrentBlockRandomness did not provide entropy    
2022-05-11 19:11:36 Running 10 warmups...    
2022-05-11 19:11:36 CurrentBlockRandomness did not provide entropy    
(removed duplicate lines)
Error: 
   0: Extrinsic is not valid: TransactionValidityError::Invalid(InvalidTransaction::BadProof)
   1: Transaction has a bad signature
```

**Review by chevdor — APPROVED** [review]
The weights in this benchmark run remain, as last time, rather stable.
I documented more small changes than usual to help test some tooling.

**Review by EgorPopelyaev — APPROVED** [review]

File: runtime/polkadot/src/weights/pallet_utility.rs [hunk 3]
```diff
@@ -45,29 +45,27 @@ use sp_std::marker::PhantomData;
...
 			.saturating_add(T::DbWeight::get().writes(1 as Weight))
 	}
 	fn dispatch_as() -> Weight {
-		(9_202_000 as Weight)
+		(9_233_000 as Weight)
 	}
 	fn force_batch(c: u32, ) -> Weight {
-		(12_678_000 as Weight)
+		(532_000 as Weight)
 			// Standard Error: 2_000
```
Comment (EgorPopelyaev) [inline runtime/polkadot/src/weights/pallet_utility.rs:68]: The weight significantly went down

**Comment by coderobe** [comment]
bot merge

File: runtime/kusama/src/weights/pallet_election_provider_multi_phase.rs [hunk 4]
```diff
@@ -105,11 +105,11 @@ impl<T: frame_system::Config> pallet_election_provider_multi_phase::WeightInfo f
...
-		(21_743_000 as Weight)
-			// Standard Error: 9_000
-			.saturating_add((1_310_000 as Weight).saturating_mul(a as Weight))
-			// Standard Error: 13_000
-			.saturating_add((190_000 as Weight).saturating_mul(d as Weight))
+		(48_969_000 as Weight)
+			// Standard Error: 10_000
+			.saturating_add((1_329_000 as Weight).saturating_mul(a as Weight))
+			// Standard Error: 16_000
+			.saturating_add((285_000 as Weight).saturating_mul(d as Weight))
```
[the remark this answers is not in the document]
Comment (ggwpez) [reply runtime/kusama/src/weights/pallet_election_provider_multi_phase.rs:112]: [swc-web link](https://weights.tasty.limo/compare?old=768bd6655577dbfda140b40fdd4ca16700fcf589&new=b6770460668a7c3ef8df0ac0f3376e1f917a17e8&threshold=10&path_pattern=runtime/**/src/weights/*.rs&method=base&ignore_errors=true#runtime/kusama/src/weights/pallet_election_provider_multi_phase.rs.elect_queued)

File: runtime/kusama/src/weights/pallet_election_provider_multi_phase.rs [hunk 5]
```diff
@@ -53,45 +53,45 @@ impl<T: frame_system::Config> pallet_election_provider_multi_phase::WeightInfo f
...
-		(10_989_000 as Weight)
+		(31_691_000 as Weight)
 			// Standard Error: 2_000
-			.saturating_add((513_000 as Weight).saturating_mul(v as Weight))
-			// Standard Error: 4_000
-			.saturating_add((71_000 as Weight).saturating_mul(t as Weight))
+			.saturating_add((517_000 as Weight).saturating_mul(v as Weight))
+			// Standard Error: 5_000
+			.saturating_add((56_000 as Weight).saturating_mul(t as Weight))
 			.saturating_add(T::DbWeight::get().writes(3 as Weight))
```
[the remark this answers is not in the document]
Comment (ggwpez) [reply runtime/kusama/src/weights/pallet_election_provider_multi_phase.rs:95]: [swc-web link](https://weights.tasty.limo/compare?old=768bd6655577dbfda140b40fdd4ca16700fcf589&new=b6770460668a7c3ef8df0ac0f3376e1f917a17e8&threshold=10&path_pattern=runtime/**/src/weights/*.rs&method=base&ignore_errors=true#runtime/kusama/src/weights/pallet_election_provider_multi_phase.rs.create_snapshot_internal)

**Comment by shawntabrizi** [comment]
@chevdor @coderobe @ggwpez @crystalin 

The ecosystem teams have asked us to mark weight updates as Runtime Noteworthy, since for some of them, it may affect their fee logic.

I have updated the tag here :)

File: runtime/kusama/src/weights/pallet_scheduler.rs [hunk 6]
```diff
@@ -62,9 +62,9 @@ impl<T: frame_system::Config> pallet_scheduler::WeightInfo for WeightInfo<T> {
...
 	// Storage: Scheduler Lookup (r:0 w:1)
 	fn on_initialize_named_resolved(s: u32, ) -> Weight {
-		(452_000 as Weight)
+		(0 as Weight)
 			// Standard Error: 29_000
-			.saturating_add((19_512_000 as Weight).saturating_mul(s as Weight))
+			.saturating_add((20_816_000 as Weight).saturating_mul(s as Weight))
 			.saturating_add(T::DbWeight::get().reads(1 as Weight))
 			.saturating_add(T::DbWeight::get().reads((2 as Weight).saturating_mul(s as Weight)))
 			.saturating_add(T::DbWeight::get().writes(1 as Weight))
```
[the remark this answers is not in the document]
Comment (shawntabrizi) [reply runtime/kusama/src/weights/pallet_scheduler.rs:70]: in situations like this, the constant factor is 2 orders of magnitude smaller than the linear factor. So even s = 1 will make this change hardly noticeable.

This is just the result of linear fitting from my view.

**Comment by chevdor** [comment]
Please note that we are reporting in this release many more than we usually do. The reason is that @ggwpez is working on a tool to help spotting changes and having a few noticeable changes helps with the debugging. 

Most of the "changes" reported in this PR are much smaller than what we usually report, for the reason mentioned above. I don't think this should be more note worthy than other releases. As a matter of fact, the weights in this PR are very close to the previous and this is, with the previous release, probably one of the least significant release in terms of weight changes.

I totally don't mind the label but it would then show up for each release because we do update the weigths (so far) for evey release. The introduction of changes such as https://github.com/paritytech/polkadot/issues/5509 however could help remove the noise and better focus on the real significant changes.

**Comment by chevdor** [comment]
FYI: As it stands now, it is unlikely that we rebuild a new RC before the final (= the current release and release notes look good :)). 

That means the new tag would not be picked up as we probably won't generate a new release notes.

I added however the task in https://github.com/paritytech/polkadot/issues/5496 to add the mention manually so it shows up.

**Pull request merged**